content/github/site-policy/github-subprocessors-and-cookies.md

```diff
@@ -41,13 +41,13 @@ When we share your information with third party subprocessors, such as our vendo
 | Zendesk | Customer support ticketing system | United States | United States |
 | Zuora | Corporate billing system | United States | United States |
 
-When we bring on a new subprocessor who handles our Users' Personal Information, or remove a subprocessor, or we change how we use a subprocessor, we will update this page. If you have questions or concerns about a new subprocessor, we'd be happy to help. Please contact us via {% data variables.contact.contact_privacy %}.
+When we bring on a new subprocessor who handles our Users’ Personal Information, or remove a subprocessor, or we change how we use a subprocessor, we will update this page. If you have questions or concerns about a new subprocessor, we’d be happy to help. Please contact us via {% data variables.contact.contact_privacy %}.
```

Comment:
Thanks for this suggestion. We follow the Microsoft style guide and use straight apostrophes and quotation marks in online content (https://docs.microsoft.com/en-us/style-guide/punctuation/quotation-marks). When I checked on this, I found that some of our newer articles use the wrong type of apostrophe and we'll look into fixing that to avoid further confusion.

```suggestion
When we bring on a new subprocessor who handles our Users' Personal Information, or remove a subprocessor, or we change how we use a subprocessor, we will update this page. If you have questions or concerns about a new subprocessor, we'd be happy to help. Please contact us via {% data variables.contact.contact_privacy %}.
```